content/bluetooth.md

```diff
@@ -15,15 +15,42 @@ section: network-troubleshooting
 tableOfContents: true
 ---
 
-## Important Notes About Bluetooth
+## About Bluetooth
 
-Bluetooth is a bit odd.
-There are a lot of factors that go into whether Bluetooth devices work together as expected.
+Bluetooth is incredibly flexible. With a range of factors that enhance compatibility, Bluetooth devices effortlessly connect and work together for a smooth, seamless experience.
 
 ### Bluetooth version
 
 Bluetooth 5.0 is backwards compatible with older Bluetooth versions, but older bluetooth versioned devices are not always compatible with newer versions or devices.
 
+### Confirm the bluetooth version of your device and make sure it is 5.0 or higher
+
+Use bluetoothctl, on your terminal type:
+
+```bash
+bluetoothctl
+```
+
+If you have multiple Bluetooth controllers, choose the one you wish to connect to the device.
+
+Check list of controllers:
+```
+List
+```
+
+Select the controller you want to use:
+
+```
+select <mac address>
+```
+
+Check the version:
+```
+version
+```
+
+![bluetoothclt version](/images/bluetooth/bluetooth_1.png)
+
 ### Signal Interference
 
 Bluetooth uses the same bandwidth as the 2.4Ghz Wi-Fi band, and in most of our machines it is on the same chip as the Wi-Fi module. They usually have two antennae, one for Bluetooth, and one for Wi-Fi, but it is possible for other Wi-Fi or Bluetooth devices signals to cross and to cause connection issues. If users are in an area crowded with other Wi-Fi networks or devices, the interference from these outside sources can impact performance and range.
```

Comment:
Just changed the wording a little bit:
Bluetooth uses the same bandwidth as the 2.4Ghz Wi-Fi band, and in some machines it is on the same chip as the Wi-Fi module. Th Wi-Fi module usually has two antennae, one for Bluetooth, and one for Wi-Fi, but it is possible for other Wi-Fi or Bluetooth devices signals to cross and to cause connection issues. If users are in an area crowded with other Wi-Fi networks or devices, the interference from these outside sources can impact performance and range.